install_wordpress.sh

```diff
@@ -0,0 +1,44 @@
+# Update and install software
```

Comment:
I would recommend to add `set -e` command at the beginning. It will stop script whenever encounting any error. Could you add in and try to recreate vm and then run `vagrant provision`